d2l-input-textarea.js

```diff
@@ -135,6 +135,11 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-input-textarea">
 				@apply --d2l-input-invalid;
 			}
 
+			:host([no-padding]) textarea.d2l-input {
+				padding-left: 0px;
+				padding-right: 0px;
+			}
```

Comment:
Agree with @dbatiste's comment about this being added to hybrid first, then re-run the P3 migration to apply it here.

In addition, `noPadding` should be added as a `bool` to the `properties` down below (with documentation). You also don't need `px` units for `0` values.